RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -144,6 +144,9 @@ namespace RockWeb.Blocks.Finance
     [BooleanField( "Enable Comment Entry", "Allows the guest to enter the the value that's put into the comment field (will be appended to the 'Payment Comment' setting)", false, "", 29 )]
     [TextField( "Comment Entry Label", "The label to use on the comment edit field (e.g. Trip Name to give to a specific trip).", false, "Comment", "", 30 )]
     [BooleanField( "Enable Business Giving", "Should the option to give as as a business be displayed", true, "", 31 )]
+    [CodeEditorField( "Invalid Fund Message", "Display this text (HTML) as an error alert if an invalid 'account' or 'glaccount' is passed through the URL.",
+        CodeEditorMode.Html, CodeEditorTheme.Rock, 200, false, "", "Text Options", 32 )]
+    [CustomDropdownListField( "Account Campus Context", "Should any context be applied to the Account List", "-1^No Account Campus Context Filter Applied,0^Only Accounts with Current Campus Context,1^Accounts with No Campus and Current Campus Context", false, "-1", "", 33 )]
 
```

Comment:
@treyhendon Can we add a [BooleanField('Allow Accounts in URL' ...] attribute that defaults to false? This might help prevent some risk of somebody specifying an account for a TransactionEntry block when we don't want them to.